app/src/components/Preferences/PreferencesCardStyles.js

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 275,
+    borderTop: '15px solid #FF4F9E',
+    margin: '1rem',
+    padding: '1rem',
+  },
+  body: {
+    margin: '0',
+    color: '#249BE5',
+  },
+  subBody: {
+    color: '#249BE5',
+  },
+  icon: {
```

Comment:
I think this `icon` class is unnecessary. I tried commenting it out and didn't see any of the styles change. Icon size should be defined in theme.js. Lmk if I'm missing something though!

Reply:
Thank you for catching that! I had no use for that.